docs/Project-Configuration/Project-and-Distro-Settings.md

```diff
@@ -65,6 +65,19 @@ Admins can also set the branch project to inherit values from a
 repo-level project settings configuration. This can be learned about at
 ['Using Repo Level Settings'](Repo-Level-Settings.md).
 
+### Project Flags
+
+Under project flags, admins have a number of options for users to configure what
+runs for their project. For example, prevent projects from creating mainline 
+commits by disabling repotracking (by default, this is enabled. Evergreen occasionally 
+misses commits due to misconfiguration or GitHub outages, in which case admins can 
```

Comment:
Perhaps we should create a repotracker settings subsection here as the latter half of this block is getting more into what the repotracker is and how the repotracker settings specifically work

Reply:
I can do this but going to leave screenshots out re comment above